README.md

```diff
@@ -5,10 +5,11 @@ This is the WIP repository for the Python version of DePSI *(), a Python package
 
 It is assumed that you have `mamba` installed. If not, you can find the installation instructions [here](https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html). Other package managers like `conda` or `venv` can be used as well.
 
-Clone this repository and `cd` into it:
+Clone this repository 
+
+Then`cd` into it:
 
 ```bash
-git clone git@github.com:TUDelftGeodesy/DePSI.git
```

Comment:
Even if a repository is private, the git clone command stays the same. You might just want to add a note saying something like: 'Before running this command, make sure you have the right permissions if the repository is private.' That way, if the repository becomes public later, the README will still make sense.

Reply:
Hi Sarah, thanks for the comment! The current repo we are looking now is a public one. The private PR is actually with a different name, so the cloning functions won't be the same. However I also do not want to have different versions of README file since that would cause trouble for synchronizing the two repo. Therefore I decide to remove it to prevent the developers on the private repo accidentally clone and develope on the public one.